nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/test/java/org/nuxeo/elasticsearch/test/TestTreeIndexing.java

```diff
@@ -398,6 +403,7 @@ public void shouldStoreOnlyEffectiveACEs() throws Exception {
     }
 
     @Test
+    @Deploy("org.nuxeo.elasticsearch.core.test:dummy-bulk-login-config.xml")
```

Comment:
If it's the same than one present in bulk or bulk test, you can reuse it as test-jar should be in classpath.

Reply:
Actually it's not the same, it has to be deployed later than the one in bulk and has a different component name, it sucks but I could not find any other way, any bright idea ?